text/src/autogluon/text/automm/configs/optimization/adamw.yaml

```diff
@@ -16,3 +16,6 @@ optimization:
   top_k_average_method: "greedy_soup"  # We support averaging method described in https://arxiv.org/pdf/2203.05482.pdf.
                                        # Currently support "uniform_soup", "greedy_soup", and "best".
   efficient_finetune: null  # Can be 'bit_fit' (only finetune bias), 'norm_fit' (finetune the normalization terms + bias terms), or null
+  loss_function: "BCEWithLogitsLoss" # The replaced loss for regresssion. Can only support loss function in torch.nn.
```

Comment:
Use an empty value by default and only specify the value in test-cases.

Reply:
Actually, we can also pick "auto" by default, which is an indicator that it should be updated by the system.